src/main/java/com/gsheet/student/service/StudentService.java

```diff
@@ -0,0 +1,123 @@
+package com.gsheet.student.service;
+
+import com.gsheet.student.dto.StudentDto;
+import com.gsheet.student.dto.google.ValueRangeUpdateDto;
+import com.gsheet.student.entity.Student;
+import com.gsheet.student.exception.StudentNotFoundException;
+import com.gsheet.student.repository.GoogleSheetRepository;
+import com.gsheet.student.resource.ValueRangeResource;
+import com.gsheet.student.transformer.ValueRangeResourceToStudentTransformer;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class StudentService {
+
+    @Autowired
+    private final GoogleSheetRepository sheetRepository;
+
+    @Autowired
+    private ValueRangeResourceToStudentTransformer sheetValueRangeDtoToStudentDtoTransformer;
+
+    public List<Student> getStudents() {
+        ValueRangeResource valueRange = this.sheetRepository.getValueRange("A:C");
+
+        List<Student> students = new ArrayList<>();
+        int index = 0;
+        for (List<String> values : valueRange.getValues()) {
+            int id = ++index;
+            students.add(sheetValueRangeDtoToStudentDtoTransformer.transform(id, values));
+        }
+
+        return students;
+    }
+
+    public Student getStudent(int id) {
+        String range = String.format("A%1$s:C%1$s", id);
```

Comment:
You can hardcoded A:C it is a bad idea, if you are planing to use the same columns on different places move it into some global static variables, or use some enums or some other aggregator to have single place with the same data